mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## v1.7.1 (June 14, 2024)
+
+### Breaking changes
+* Mountpoint now only accepts KMS key ARN as a valid input for `--sse-kms-key-id` argument. This prevents an issue where Mountpoint could crash when uploading a file with KMS key identifiers other than the key ARN. ([#908](https://github.com/awslabs/mountpoint-s3/pull/908))
```

Comment:
Let's be a bit more intentional of explaining the breaking change, with a little more context.

```suggestion
* Mountpoint supports specifying KMS server-side encryption for new object uploads using the key ARN only. Mountpoint now validates the value provided with `--sse-kms-key-id` at mount time. This prevents an issue where Mountpoint could crash when uploading a file with KMS key identifiers other than the key ARN. ([#908](https://github.com/awslabs/mountpoint-s3/pull/908))
```